src/views/Orders.vue

```diff
@@ -282,7 +319,7 @@ export default defineComponent({
         });
 
       return jobAlert.present();
-    }
+    },
```

Comment:
Remove unwanted change

Reply:
Okay sir!

Reply:
@ymaheshwari1 sir, I've made all the changes as said, Please review
https://github.com/hotwax/job-manager/pull/140/commits/5b4e8884f2999fb2614dcd5679e0cbb6622cdfee